### ISSUE TITLE
Minetest 0.4.11 Crashes When You Click Anything

### ISSUE THREAD

**Issue by Heybye44** [minetest/minetest#2315]
So far I have only tested this on one of my PCs. Minetest hangs for a few seconds then crashes after clicking a button. Minecraft 1.8 has a similar problem so it might just be my computer. 

Here's all I get from debug.txt

```
Irrlicht log: Irrlicht Engine version 1.8.1 
Irrlicht log: Microsoft Windows XP Professional Service Pack 3 (Build 2600) 
Irrlicht log: Using renderer: OpenGL 1.4.5646 
Irrlicht log: RADEON XPRESS 200 Series SW TCL x86/SSE2: ATI Technologies Inc. 
Irrlicht log: OpenGL driver version is 1.2 or better. 
Irrlicht log: GLSL version: 1.1 
Irrlicht log: Resizing window (800 600)
```

Minetest has been rendered unplayable. ;o;

**Comment by Zeno-** [comment]
Hi,

Can you please try one of the builds here: https://forum.minetest.net/viewtopic.php?f=18&t=11227

and see if the problem is still present?

**Comment by Heybye44** [comment]
![screenshot 2015-02-14 10 12 42](https://cloud.githubusercontent.com/assets/6135977/6200026/835fd502-b432-11e4-9b75-e7d9d7c05e44.png)
Google Chrome reports that this file is malicious.

**Comment by BlockMen** [comment]
Chrome is known to do that sometimes. (see https://forum.minetest.net/viewtopic.php?p=105487#p105487)
It tends to warn more often of zip-files, Idk why :\

And since I made that builds I can guarantee you its safe.

**Comment by SmallJoker** [comment]
You can go around that problem by clicking on the `Down (v)` Button at the right of `Dismiss`.
There you can allow to keep the file.

**Comment by Heybye44** [comment]
The release candidate crashes as soon as you click anything. The "say sorry for the inconvenience"
popup does not even appear.

**Comment by SmallJoker** [comment]
Open debug.txt and paste the latest 10 lines here in a code box.

**Comment by Heybye44** [comment]
<code>-------------
  Separator  
-------------

Irrlicht log: Irrlicht Engine version 1.8.1
Irrlicht log: Microsoft Windows XP Professional Service Pack 3 (Build 2600)
Irrlicht log: Using renderer: OpenGL 1.4.5646
Irrlicht log: RADEON XPRESS 200 Series SW TCL x86/SSE2: ATI Technologies Inc.
Irrlicht log: OpenGL driver version is 1.2 or better.
Irrlicht log: GLSL version: 1.1
Irrlicht log: Resizing window (800 600)
</code>

**Comment by SmallJoker** [comment]
Open or create the file minetest.conf in the subdirectory of minetest.exe
Add or change the setting `video_driver = software` save and retry.

**Comment by Heybye44** [comment]
That didn't work.

**Comment by BlockMen** [comment]
You said it crashes clicking a button. What buttons you tried? Also, does it crash when you click something else, like a tab? What happens when you double-click outside the menu (blue background)?

**Comment by Heybye44** [comment]
Double clicking on the background crashed my graphics card/computer.
(Everything froze the screen then went black and the fans got quieter)

**Comment by BlockMen** [comment]
Can you reproduce that with Minetest 0.4.10 (msvc)?

**Comment by SmallJoker** [comment]
Try to update your graphics driver, looks like it has troubles with rendering stuff.

**Comment by Heybye44** [comment]
I've been sc=watching for a update and haven't found one. I'm pretty sure my graphics card is discontinued.



—

**Comment by rubenwardy** [comment]
It would be useful for you to try 0.4.10 to see if the problem is there as well. Narrows it down about.

**Comment by Heybye44** [comment]
0.4.10 Crashes when you double click on the background, but is
perfectly playable.

**Comment by SmallJoker** [comment]
That double click crash has been added for smartphones to exit a formspec.
I guess that feature should be ignored for desktop users.

**Comment by BlockMen** [comment]
@SmallJoker The idea to test that was to check if its a general problem with mouse input via Irrlicht, not because its a "feature" ;)

@Heybye44 sorry, i have currently no idea what could cause this.

**Comment by Heybye44** [comment]
It's fixed in 0.4.12
